lib/tmdb/client.rb

```diff
@@ -35,7 +35,7 @@ def build_url(endpoint, params)
         when :tv_season_data then "/tv/#{params[:series_id]}/season/#{params[:season_number]}?api_key=#{API_KEY}&append_to_response=credits"
         when :tv_episode_data then "/tv/#{params[:series_id]}/season/#{params[:season_number]}/episode/#{params[:episode_number]}?api_key=#{API_KEY}"
         when :multi_search then "/search/multi?api_key=#{API_KEY}&query=#{searchable_query(params[:query])}"
-        when :streaming_service_providers then "/movie/#{params[:movie_id]}/watch/providers?api_key=#{API_KEY}"
+        when :streaming_service_providers then "/#{params[:media_type]}/#{params[:tmdb_id]}/watch/providers?api_key=#{API_KEY}&watch_region=US"
```

Comment:
This `watch_region` argument does not seem to work. 🤔 I'm thinking I should probably remove it. What do you think?

Reply:
🤔 I'll defer to you since I haven't looked at the api in quite a while and I'm not sure how you are determining whether it works or not. I'm happy to discuss, but off hand, dunno.